src/tests/functional/shared_test_classes/src/base/ov_subgraph.cpp

```diff
@@ -243,6 +245,25 @@ void SubgraphBaseTest::infer() {
     inferRequest = compiledModel.create_infer_request();
     for (const auto& input : inputs) {
         inferRequest.set_tensor(input.first, input.second);
+
+        // debug
+        if (getenv("DUMP_INPUTS")) {
+            const auto& input_tensor = input.second;
+            std::cout << "dump input " << input.first->get_friendly_name() << ", with shape " << input_tensor.get_shape() << std::endl;
+            switch (input_tensor.get_element_type()) {       
+            case ElementType::i64: 
+            {
+                const auto inputData = input_tensor.data<ov::element_type_traits<ElementType::i64>::value_type>();
+                for (auto i = 0; i < input_tensor.get_size(); i++) std::cout << "index: "<< i << ": " << inputData[i] << std::endl;
+                break;
+            }
+            default:
+            {
+                const auto inputData = input_tensor.data<ov::element_type_traits<ElementType::f32>::value_type>();
+                for (auto i = 0; i < input_tensor.get_size(); i++) std::cout << "index: "<< i << ": " << inputData[i] << std::endl;
+            }     
+            }
+        }
```

Comment:
Better to implement is scope of separate PR if it supposed to be kept

Reply:
removed.